cmd/add.go

```diff
@@ -22,15 +22,19 @@ var addCmd = &cobra.Command{
 }
 
 var noEncrypt bool
+var noClean bool
 
 func init() {
 	RootCmd.AddCommand(addCmd)
 
 	addCmd.Flags().BoolVarP(&noEncrypt, "no-encrypt", "n", false, "Do not encrypt the file after adding")
+	addCmd.Flags().BoolVar(&noClean, "no-clean", false, "Do not clean up plaintext after encrypting")
 }
 
 // AddCommand the command for the add command
 func AddCommand(cmd *cobra.Command, args []string) error {
```

Comment:
CodeClimate: Function AddCommand has 5 return statements (exceeds 4 allowed).

I'm unsure how to fix this one

Reply:
yeah... with go it seems like you have to do a ton of guard work, checking errors throughout. idk what the community thinks about this kinda thing, but I'd be OK turning that check off, it seems somewhat arbitrary in this case.